nntrainer/layers/fc_layer.cpp

```diff
@@ -33,103 +33,94 @@ namespace nntrainer {
 
 enum FCParams { weight, bias };
 
-int FullyConnectedLayer::initialize(Manager &manager) {
+void FullyConnectedLayer::finalize(InitLayerContext &context) {
   auto unit = std::get<props::Unit>(fc_props).get();
-  int status = ML_ERROR_NONE;
 
-  if (getNumInputs() != 1) {
+  if (context.getNumInputs() != 1) {
```

Comment:
@zhoonit Is it ok for CTC of TIZEN if we remove the return value?

Reply:
Yes, our api captures throw and convert back as an integer.